src/test/java/api/requests/RequestsAPIMoveTests.java

```diff
@@ -26,6 +32,18 @@
 
 @RunWith(JUnitParamsRunner.class)
 public class RequestsAPIMoveTests extends APITests {
```

Comment:
What is the difference between the `RequestsAPIMoveTests` and `MoveRecallRequestTests`? Could the former, be moved to `scenarios/MoveRequestTests`?

Reply:
I don’t have an answer to that. I can move them all to scenarios and comment referencing the issue scenario they are testing.

Reply:
> I don’t have an answer to that. I can move them all to scenarios and comment referencing the issue scenario they are testing.

I think the move make senses.

I prefer the names of the tests to be sufficiently descriptive to avoid the need for external references where possible. And where not, to describe the scenario with the code (though that tends to be a signal to me that the design of the system isn't expressive enough)